app/Listeners/SendTaggedNotification.php

```diff
@@ -36,7 +36,7 @@ public function handle(PostTagged $event)
             ]);
         }
 
-        if ($tag->tag_slug === 'good-for-storytelling') {
+        if (! ($post->hasGoodTag()) && in_array($tag->tag_slug, ['good-for-storytelling', 'good-submission', 'good-for-sponsor', 'good-quote'])) {
```

Comment:
good catch @deguzmankevin ! I was wondering about this too when we were talking about notifying Slack for other tags as well. I love this solution so it will only notify the channel once!